### ISSUE TITLE
Extend PlaceholderAPI support

### ISSUE THREAD

**Issue by diogoxarons** [PyvesB/AdvancedAchievements#295]
Hello .
This plugin have now a **PlaceholderAPI support*** and that´s great, but, it would be great have more useful placeholders .

For example:
1. Total unlocked achievements (number)
2. Total unlocked achievements (%)
3. Total achievements (number)

That would be great .. really useful in scoreboard .

Thank you .

***List of Placeholders** : https://github.com/PyvesB/AdvancedAchievements/pull/269

**Comment by PyvesB** [comment]
Hello @diogoxarons ,

@Phoetrix implemented PlaceholderAPI support in this project, maybe he'll be interested in taking a look at these new features as well? 😃 

Cheers,

Pyves

**Comment by diogoxarons** [comment]
I hope yes ! :)

Thank you .

**Comment by Phoetrix** [comment]
Yes it's a good idea! :)
Unfortunately I have exams from 18th till 29th of January so I will probably do that at the end of the month.

Thank you for your patience! :)

**Comment by diogoxarons** [comment]
@Phoetrix 
Waiting for news :)

Thank you .

**Comment by Phoetrix** [comment]
Hi @PyvesB !

I have a problem. I have finish to add these three news placeholders but I can't do a build. I don't know why.
Error messages don't help me because I am not very familiar with maven and eclipse. :(
I send to you the error folder.

[surefire-reports.zip](https://github.com/PyvesB/AdvancedAchievements/files/1670244/surefire-reports.zip)

**Comment by Phoetrix** [comment]
And I understand nothing about the website of Github. You say to me on my last pull request that I need to keep my master branch as an exact copy of the main repository but how?

The website say to me that I have conflicts with some files and "_Merging is not possible because you have unmerged files._"
But.... I just want to replace my current copy of your repository with the latest version of your repository. Github is really boring.. :((

I searched on Internet and I saw a list of commands to execute on a console window. I understand nothing about that and why is there not a simpler way like a user-friendly interface? ._.

So I think that I can delete my repository and re-click on the "Fork" button? It will work? Because I'm very lost, I am really sorry... :(

**Comment by PyvesB** [comment]
Hey @Phoetrix! 

Are you trying to build via Eclipse? Is Eclipse configured to use a Java JDK and not just a JRE (check in _Preferences -> Java -> Installed JREs_, this should actually point to a JDK installation)?

Regarding GitHub, I don't know where you have made your new changes, but store them somewhere secure on your computer and try the following commands on your master branch:
```
git fetch upstream
git reset --hard upstream/master
git push origin master --force
```

Then create a new branch, for instance named "placeholderapi":
```
git checkout -b placeholderapi
```

Apply your previously saved changes to the project and commit to that branch. You should then be able to use the GitHub UI to submit the pull request. 😉 

Let me know whether this helps!

**Comment by Phoetrix** [comment]
Yes I build via Eclipse. Everything seems good for me but the build doesn't work. The only way that I found to build is to add these followings lines on the _pom.xml_:

`<plugin>
   <groupId>org.apache.maven.plugins</groupId>
   <artifactId>maven-surefire-plugin</artifactId>
   <version>2.19.1</version>
   <configuration>
      <testFailureIgnore>true</testFailureIgnore>
   </configuration>
</plugin>`

And thank you for your help about Github. I will send my pull request. ^^

**Comment by PyvesB** [comment]
If you still have build issues even after ensuring that you are running a JDK, you can try creating a new Maven Build configuration and put `clean install -DskipTests` as goals. To create a new Maven Build configuration, right click on the project, _Run as-> Run configurations... -> Maven Build -> New launch configuration_. That should skip the tests entirely without having to modify the _pom.xml_. ^^

**Comment by PyvesB** [comment]
@diogoxarons: here's a test version containing @Phoetrix's implementation! You can refer to [his Wiki page](https://github.com/PyvesB/AdvancedAchievements/wiki/Placeholders) for more information about the different Placeholders. If everything goes well, I'll try to release this Thursday evening or otherwise next week.

[AdvancedAchievements.zip](https://github.com/PyvesB/AdvancedAchievements/files/1679115/AdvancedAchievements.zip)

**Comment by diogoxarons** [comment]
Thank you very much !

**Comment by PyvesB** [comment]
I have released this as version 5.6.3. Good work everyone!
